cmd/gokr-packer/write.go

```diff
@@ -178,6 +180,70 @@ var (
 	}
 )
 
+type kernelManifest struct {
+	KernelGlobs     []string `toml:"kernel_globs"`
+	UseMBR          bool     `toml:"use_mbr"`
```

Comment:
Instead of `UseMBR`, I suggest `WriteMBROnlyWithoutGPT`. It’s long, but descriptiveness is an advantage in this intricate code base :)